src/main/scala/org/opennetworkinsight/proxy/ProxySuspiciousConnectsModel.scala

```diff
@@ -172,7 +172,7 @@ object ProxySuspiciousConnectsModel {
                         entropyCuts: Array[Double],
                         agentCuts: Array[Double]): RDD[OniLDACInput] = {
 
-    val topDomains: Broadcast[Set[String]] = sc.broadcast(TopDomains.TOP_DOMAINS)
+    val topDomains: Broadcast[Set[String]] = sc.broadcast(TopDomains.TopDomains)
 
     val agentToCountBC = sc.broadcast(agentToCount)
     val udfWordCreation = ProxyWordCreation.udfWordCreation(topDomains, agentToCountBC, timeCuts, entropyCuts, agentCuts)
```

Comment:
@NathanSegerlind I don't want to be too picky but seems like a new convention was defined during this restructure. Looks like udf functions are called like `functionNameUDF` but in this line, I can still see the old fashion naming.